share/sysbuild/CMakeLists.txt

```
@@ -35,7 +35,9 @@ get_property(IMAGES GLOBAL PROPERTY sysbuild_images)
 sysbuild_module_call(PRE_CMAKE MODULES ${SYSBUILD_MODULE_NAMES} IMAGES ${IMAGES})
 sysbuild_images_order(IMAGES_CONFIGURATION_ORDER CONFIGURE IMAGES ${IMAGES})
 foreach(image ${IMAGES_CONFIGURATION_ORDER})
+  sysbuild_module_call(PRE_IMAGE_CMAKE MODULES ${SYSBUILD_MODULE_NAMES} IMAGES ${IMAGES} IMAGE ${image})
   ExternalZephyrProject_Cmake(APPLICATION ${image})
+  sysbuild_module_call(POST_IMAGE_CMAKE MODULES ${SYSBUILD_MODULE_NAMES} IMAGES ${IMAGES} IMAGE ${image})
```

`IMAGE` doesn't seems to be known / defined argument to `sysbuild_module_call()`, so if you want things to work, then you should either add `IMAGE` to list of known and processed args, or if the `IMAGE` arg is intended for the hook implementation, then update the call to the following:
```suggestion
  sysbuild_module_call(POST_IMAGE_CMAKE MODULES ${SYSBUILD_MODULE_NAMES} IMAGES ${IMAGES} EXTRA_ARGS IMAGE ${image})
```

same comment applies to the pre-hook.

---

Fixed